### PR TITLE
WIKI-1335 : do not allow to rename a page to the name of an existing page

### PR DESCRIPTION
**Pull request by thomasdelhomenie**
Do not allow to rename a page to the name of an existing page.

### CODE REVIEW

File: wiki-webui/src/main/java/org/exoplatform/wiki/webui/UIFieldEditableForm.java [hunk 1]
```diff
@@ -173,21 +173,12 @@ public void processEvent(Event<UIFieldEditableForm> event) throws Exception {
...
 
-      try {
-        WikiNameValidator.validate(titleInput.getValue());
-      } catch (IllegalArgumentException ex) {
-        isError = true;
-        Object[] arg = { ex.getMessage() };
-        appMsg = new ApplicationMessage("WikiPageNameValidator.msg.Invalid-char",
-                                          arg,
-                                          ApplicationMessage.WARNING);
-      }
```
Comment (trongtt) [inline wiki-webui/src/main/java/org/exoplatform/wiki/webui/UIFieldEditableForm.java]: could you explain why do you remove these try catch block ?

Comment (thomasdelhomenie) [inline wiki-webui/src/main/java/org/exoplatform/wiki/webui/UIFieldEditableForm.java]: This change is actually not directly related to this issue. We removed the check on title when creating or updating a page in a previous issue but we did not remove the check when renaming only (for example, try to create or update a page with the character @, it works, then try to rename a page and add a @, it is not allowed). So I removed this check to have a consistent behavior.

Comment (trongtt) [inline wiki-webui/src/main/java/org/exoplatform/wiki/webui/UIFieldEditableForm.java]: Yes, that's what I guess. Thank you.
